File: tcv2.Api/Data/Entities/ChordSheet.cs
using System;
using System.ComponentModel.DataAnnotations;


namespace tcv2.Api.Data.Entities
{
    public class ChordSheet
    {
        [Key]
        public Guid Id { get; set; }

        public Guid? OrgId { get; set; }

        public string? Title { get; set; }

        public string? Artist { get; set; }

        public string? Content { get; set; }

        public string? Key { get; set; }

        public DateTime CreatedAt { get; set; }

        public DateTime? UpdatedAt { get; set; }

        // Navigation properties
        public Organization? Organization { get; set; }
    }
}

File: mobile/lib/models/chord_sheet.dart
class ChordSheet {
  final String id;
  final String title;
  final String artist;
  final String content;
  final String key;

  ChordSheet({required this.id, required this.title, required this.artist, required this.content, required this.key});

  factory ChordSheet.fromJson(Map<String, dynamic> json) {
    return ChordSheet(
      id: json['id']?.toString() ?? '',
      title: json['title'] ?? json['Title'] ?? '',
      artist: json['artist'] ?? json['Artist'] ?? '',
      content: json['content'] ?? json['Content'] ?? '',
      key: json['key'] ?? json['Key'] ?? '',
    );
  }
}
